### ISSUE TITLE
Unable to complete the build in win10+VScode

### ISSUE THREAD

**Issue by JayQu9** [Open-Agriculture/AgIsoVirtualTerminal#34]
win10
Visual Studio Code: Version: 1.95.1 (user setup)
cmake version 3.28.0-rc5
mingw64: I tried three versions and none of them worked
(1)gcc version 7.3.0 (x86_64-posix-seh-rev0, Built by MinGW-W64 project)
(2)gcc version 8.1.0 (x86_64-posix-seh-rev0, Built by MinGW-W64 project)
(3)gcc version 14.1.0 (x86_64-posix-seh-rev0, Built by MinGW-Builds project)

this is build log:
`
[proc] Executing command: "C:\Program Files\CMake\bin\cmake.EXE" --build d:/AgIsoVirtualTerminal/build --config Debug --target all -j 10 --
[build] [  2%] Built target AgISOVirtualTerminal_rc_lib
[build] [  7%] Built target Utility
[build] [ 49%] Built target Isobus
[build] [ 50%] Building CXX object _deps/can_stack-build/hardware_integration/CMakeFiles/HardwareIntegration.dir/src/sys_tec_windows_plugin.cpp.obj
[build] In file included from D:/AgIsoVirtualTerminal/build/_deps/can_stack-src/hardware_integration/include/isobus/hardware_integration/sys_tec_windows_plugin.hpp:23,
[build]                  from D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:12:
[build] D:/AgIsoVirtualTerminal/build/_deps/can_stack-src/hardware_integration/include/isobus/hardware_integration/Usbcan32.h:787:1: warning: multi-line comment [-Wcomment]
[build]  //#define USBCAN_CALC_TIMEDIFF(old_time,new_time) \
[build]  ^
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp: In member function 'virtual void isobus::SysTecWindowsPlugin::open()':
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:69:47: warning: missing initializer for member '_tUcanInitCanParam::m_bMode' [-Wmissing-field-initializers]
[build]      tUcanInitCanParam initCANParameters = { 0 };
[build]                                                ^
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:69:47: warning: missing initializer for member '_tUcanInitCanParam::m_bBTR0' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:69:47: warning: missing initializer for member '_tUcanInitCanParam::m_bBTR1' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:69:47: warning: missing initializer for member '_tUcanInitCanParam::m_bOCR' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:69:47: warning: missing initializer for member '_tUcanInitCanParam::m_dwAMR' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:69:47: warning: missing initializer for member '_tUcanInitCanParam::m_dwACR' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:69:47: warning: missing initializer for member '_tUcanInitCanParam::m_dwBaudrate' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:69:47: warning: missing initializer for member '_tUcanInitCanParam::m_wNrOfRxBufferEntries' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:69:47: warning: missing initializer for member '_tUcanInitCanParam::m_wNrOfTxBufferEntries' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:40: warning: missing initializer for member '_tUcanChannelInfo::m_bMode' [-Wmissing-field-initializers]
[build]      tUcanChannelInfo channelInfo = { 0 };
[build]                                         ^
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:40: warning: missing initializer for member '_tUcanChannelInfo::m_bBTR0' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:40: warning: missing initializer for member '_tUcanChannelInfo::m_bBTR1' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:40: warning: missing initializer for member '_tUcanChannelInfo::m_bOCR' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:40: warning: missing initializer for member '_tUcanChannelInfo::m_dwAMR' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:40: warning: missing initializer for member '_tUcanChannelInfo::m_dwACR' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:40: warning: missing initializer for member '_tUcanChannelInfo::m_dwBaudrate' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:40: warning: missing initializer for member '_tUcanChannelInfo::m_fCanIsInit' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:40: warning: missing initializer for member '_tUcanChannelInfo::m_wCanStatus' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:70:22: warning: variable 'channelInfo' set but not used [-Wunused-but-set-variable]
[build]      tUcanChannelInfo channelInfo = { 0 };
[build]                       ^~~~~~~~~~~
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp: In member function 'virtual bool isobus::SysTecWindowsPlugin::read_frame(isobus::CANMessageFrame&)':
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:108:35: warning: missing initializer for member '_tCanMsgStruct::m_bFF' [-Wmissing-field-initializers]
[build]     tCanMsgStruct CANMessage = { 0 };
[build]                                    ^
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:108:35: warning: missing initializer for member '_tCanMsgStruct::m_bDLC' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:108:35: warning: missing initializer for member '_tCanMsgStruct::m_bData' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:108:35: warning: missing initializer for member '_tCanMsgStruct::m_dwTime' [-Wmissing-field-initializers]
[build] D:\AgIsoVirtualTerminal\build\_deps\can_stack-src\hardware_integration\src\sys_tec_windows_plugin.cpp:130:10: error: 'std::this_thread' has not been declared
[build]      std::this_thread::sleep_for(std::chrono::milliseconds(1)); // This is long enough for 2 messages to be received at max bus load, which should be fine.
[build]           ^~~~~~~~~~~
[build] mingw32-make.exe[2]: *** [_deps\can_stack-build\hardware_integration\CMakeFiles\HardwareIntegration.dir\build.make:151: _deps/can_stack-build/hardware_integration/CMakeFiles/HardwareIntegration.dir/src/sys_tec_windows_plugin.cpp.obj] Error 1
[build] mingw32-make.exe[1]: *** [CMakeFiles\Makefile2:326: _deps/can_stack-build/hardware_integration/CMakeFiles/HardwareIntegration.dir/all] Error 2
[proc] The command: "C:\Program Files\CMake\bin\cmake.EXE" --build d:/AgIsoVirtualTerminal/build --config Debug --target all -j 10 -- exited with code: 2
[build] mingw32-make.exe: *** [Makefile:155: all] Error 2
[driver] Build completed: 00:00:02.608
[build] Build finished with exit code 2

`

**Comment by JayQu9** [comment]
I tried your latest fix, but it still reported an error. At this time, you only need to comment out a driver in CMAKE
（list(APPEND CAN_DRIVER "WindowsInnoMakerUSB2CAN")）. It can be compiled, but can't run. It seems that needs to be compiled into an installation package for use, just like MSVC.
 I changed to the MSVC compilation environment. The directly compiled exe cannot be run directly. The generated installation package can be used after installation. Anyway, the first step was successful.
